README.md

```diff
@@ -716,51 +692,18 @@ not relevant to the function, there is no need to subscribe to it. The following
 the `runtimeConfig` refreshable: 
 
 ```go
-myNumRefreshable := runtimeConfig.Map(func(in interface{}) interface{} {
-    return in.(AppRuntimeConfig).MyNum
+myNumRefreshable, _ := refreshable.Map(runtimeConfig, func(in AppRuntimeConfig) int {
```

Comment:
I think this documentation should mention the second return value (and/or there should be a comment on this example code to explain why it is being dropped/ignored)